### PR TITLE
Add Cloud Base Altitude to Weather Display

### LINKED ISSUE #1061: [UI] Add Cloud Base Altitude to Weather Display
The new clouds can have a big effect on weapons selection and employment. Adding the height of the cloud base to the UI will allow players to make better decisions when designing their strike packages and selecting their weapons loadouts.

### PR DESCRIPTION
**Pull request by nosv1**
https://github.com/dcs-liberation/dcs_liberation/issues/1061

Couldn't really think of an ideal implementation of this. A tooltip was a 2nd last resort beyond not being able to find a good spot to put this information in the weather box, the last resort was having like a little button thing in the bottom right corner of the weather box and clicking it would expand it to give more information, but that's a bit over my head atm.

As for the code, I think this is the most right way to write it? lmk.

image

doesn't appear to convert exactly into DCS, but in both my tests it was respectivley close 

image

### CODE REVIEW

**Comment by HILOK** [comment]
what about replacing the wx info altogether with proper METAR? maybe as option? the tooltips within the DCS ME give a good description of clouds. also ceilings in METAR reports are rounded to hundrets, which resolves the slight conversion error above. wind would have to be reciprocal in order to comply with western reports. the rest is copy paste, i guess. dew point could be calculated as function of ceiling, if one wanted to get fancy.

**Comment by Mustang-25** [comment]
Yeah, I agree with HILOK. It would probably be more beneficial to just replace it with the METAR. I haven't really tested it a lot to see if the METAR lines up with the cloud base though so it might still need that extra tooltip.

**Comment by HILOK** [comment]
thanks. good point! the DCS preset tooltips might be fixed to standard values. so the correct ceiling -if changed by liberation- would have to be extracted using nosv1's method in order to be included in the METAR report.

File: qt_ui/widgets/QTopPanel.py [hunk 1]
```diff
@@ -112,6 +112,11 @@ def setGame(self, game: Optional[Game]):
         self.transfers.setEnabled(True)
 
         self.conditionsWidget.setCurrentTurn(game.turn, game.conditions)
+
+        self.conditionsWidget.setToolTip(
+            f"Cloud Base: {game.conditions.weather.clouds.base}m / {round(game.conditions.weather.clouds.base * 3.281)}ft"
```
Comment (bgreman) [inline qt_ui/widgets/QTopPanel.py]: Instead of hardcoding this conversion factor, consider using:

`game.utils.METERS_TO_FEET`

Comment (nosv1) [reply qt_ui/widgets/QTopPanel.py]: image

regarding this (the pic): won't this break stuff? like i figure .METERS_TO_FEET would be a 'good enough' solution without breaking anything

perhaps tho, i implemented it incorrectly in commit `base is now Distance`

**Review by bgreman — APPROVED** [review]

**Pull request merged**